domains/ypratham.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Pratham Yadav Portfolio",
+  "repo": "https://github.com/ypratham/portfolio.github.io",
+  "owner": {
+    "username": "ypratahm",
+    "email": "ypratham0014@gmail.com"
+  },
+  "record": {
+    "CNAME": "portfolio.github.io"
```

Comment:
This has to be `ypratham.github.io`. You can have a look at [github pages docs](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-github-pages-site) or ask your questions on [github discuss](https://github.com/is-a-dev/register/discussions) if you need any help.